app/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "io.github.prices.crawler.mobile"
```

Comment:
this does not support dashes?
because the organization domain is: prices-crawler.github.io